app/gradlew

```diff
@@ -0,0 +1,240 @@
+#!/bin/sh
```

Comment:
Should this be really added to the commit? I think we might have some issue with .gitignore file.

Reply:
I think merging the main branch into your branch will help with that